tsl/src/continuous_aggs/create.c

```diff
@@ -383,18 +385,17 @@ cagg_add_trigger_hypertable(Oid relid, char *trigarg)
 		foreach (cell, ht->data_nodes)
 		{
 			HypertableDataNode *node = lfirst(cell);
-			StringInfo command = makeStringInfo();
-			appendStringInfo(command,
-							 "CREATE TRIGGER %s AFTER INSERT OR UPDATE OR DELETE ON %s.%s FOR EACH "
-							 "ROW EXECUTE FUNCTION %s.%s(%d, %d)",
-							 quote_identifier(CAGGINVAL_TRIGGER_NAME),
-							 quote_identifier(NameStr(ht->fd.schema_name)),
-							 quote_identifier(NameStr(ht->fd.table_name)),
-							 quote_identifier(INTERNAL_SCHEMA_NAME),
-							 quote_identifier(CAGG_INVALIDATION_TRIGGER),
-							 node->fd.node_hypertable_id, /* distributed member hypertable ID */
-							 node->fd.hypertable_id /* Access Node hypertable ID */);
-			cmd_descr_data[i].sql = command->data;
+			char node_hypertable_id_str[12];
+			CreateTrigStmt remote_stmt = stmt_template;
+
+			/* 1st argument: distributed member hypertable ID */
+			pg_ltoa(node->fd.node_hypertable_id, node_hypertable_id_str);
+			/* 2nd argument: Access Node hypertable ID */
+			pg_ltoa(node->fd.hypertable_id, hypertable_id_str);
```

Comment:
Comments just duplicate code, so they are not needed.
```suggestion
			pg_ltoa(node->fd.node_hypertable_id, node_hypertable_id_str);
			pg_ltoa(node->fd.hypertable_id, hypertable_id_str);
```